### ISSUE TITLE
how do i get my PR merged?

### ISSUE THREAD

**Issue by npomfret** [ccxt/ccxt#4008]

**Comment by kroitor** [comment]
https://github.com/ccxt/ccxt/pull/3965

It requires a parser for https://quoine.zendesk.com/hc/en-us/articles/360004828252-Minimum-withdrawal-amount-Minimum-order-quantity to be rewritten as we've deprecated hardcoding. Hope to rewrite it asap.

**Comment by npomfret** [comment]
@kroitor thanks for the response.

There were already hard coded values in there for min order sizes which were incorrect.  I have fixed them and added more.

I appreciate you would prefer to parse the web page, but can this not be done as a future enhancement?

There are lots of other fixes in here.  I needed to make them as the implementation was too buggy for me to use to trade with.  I think they're all good (I am trading now with my fork), and would like to do more but am hesitant to do so until these have been accepted.
